apps/stripe/src/app/api/saleor/payment-gateway-initialize-session/response-params.ts

```diff
@@ -0,0 +1,9 @@
+import { z } from "zod";
+
+export const responseData = z
+  .object({
+    stripePublishableKey: z.string(),
+  })
+  .brand("PaymentGatewayInitializeSessionResponseData");
+
+export type ResponseDataType = z.infer<typeof responseData>;
```

Comment:
we dont have any protected against importing incorrect stuff, and probably we will have as many "ResponseDataType" as many routes we will have

it makes poor DX because Typescript will import *one of 5 ResponseDataType* - no necessarily from `./`

I suggest either:
1. Make export names explicit like `PaymentGatewayInitializeSessionResponseDataType`
2. Or setup dependency cruiser to ensure file from route folder can't import file from another route (which iMO we should do anyway!)